app/components/audio/song-control.tsx

```
@@ -1,57 +1,154 @@
+            </p>
+            <input 
+              className={twMerge(
+                "w-full",
+                isDisabled && "opacity-50"
+              )}
+              aria-label="song-progress" 
+              type="range" 
+              value={isEmpty ? 0 : progress} 
+              min={0} 
+              max={currentSong?.duration || 100} 
+              onChange={(e) => seekTo(parseInt(e.target.value))}
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)